testing/a11y/src/Component.cy.tsx

```diff
@@ -6,6 +6,24 @@ import * as React from 'react';
 import * as allExamples from 'examples';
 import { ThemeProvider } from '@itwin/itwinui-react';
 
+type commonRulesKeys = 'ignore_color_contrast_violation';
+const commonRules: Record<commonRulesKeys, Record<string, any>> = {
+  ignore_color_contrast_violation: {
+    id: 'color-contrast',
+    enabled: true,
+    selector: ':not(.iui-root, [data-iui-theme]) .iui-time > ol > li)',
```

Comment:
Should the closing parenthesis of the `:not` be after the `li`?

Because I believe the logic of this L13-14 is to enable the `color-contrast` rule for all tags that match the selector. So because of the `:not( ... li)`, it means the rule is enabled for all tags that are *not* those `li`s.

Or maybe, can you explain your reasoning for that selector?

Reply:
> I was unsure about the right path to get to the `li` element. So I kind of followed my way as shown in the screenshot.

I see. That CSS selector might also work. But still the `:not()` would likely need to encapsulate the entire selector:

```diff
- selector: ':not(.iui-root, [data-iui-theme]) .iui-time > ol > li)',
+ selector: ':not(:where(.iui-root, [data-iui-theme]) .iui-time > ol > li)', 
```

> Also, I did this because I thought we only wanted to select those `li` elements that are descendants of `ol` and are children of class `iui-time`.

Yes, the goal is to select *only* the `li` in `ol` in `iui-time`. So a more specific selector is ideal here. 

> My logic will exclude the selecting of `li` elements which are descendants of class `iui-root` with data attribute `data-iui-theme`.
> 
> Not sure if this correct.

I think just enabling the rule for `.iui-time > ol > li` would be enough. The `.iui-root, [data-iui-theme]` may not be needed. But I haven't tested this. So you can confirm it.